src/api.v2/controllers/metadataTrackController.js

```diff
@@ -0,0 +1,76 @@
+const MetadataTrack = require('../model/MetadataTrack');
+
+const getLogger = require('../../utils/getLogger');
+const { OK, NotFoundError } = require('../../utils/responses');
+
+const logger = getLogger('[MetadataTrackController] - ');
+
+const createMetadataTrack = async (req, res) => {
+  const {
+    params: { experimentId, metadataTrackKey },
+  } = req;
+
+  logger.log(`Creating metadata track ${metadataTrackKey} in experiment ${experimentId}`);
+
+  await new MetadataTrack().createNewMetadataTrack(experimentId, metadataTrackKey);
+
+  logger.log(`Finished creating metadata track ${metadataTrackKey} in experiment ${experimentId}`);
+
+  res.json(OK());
+};
+
+const patchMetadataTrack = async (req, res) => {
+  const {
+    params: { experimentId, metadataTrackKey: oldKey },
+    body: { key },
+  } = req;
+
+  logger.log(`Patching metadata track ${oldKey} in experiment ${experimentId}`);
+
+  const result = await new MetadataTrack()
+    .update({ experiment_id: experimentId, key: oldKey }, { key });
```

Comment:
I see that `patchMetadataTrack` updates the `oldKey` to `newKey` and from what I understand by reading the rest of the code, the key is not equivalent to the value of metadata (for example, the initial and default value of `N.A.` is not key). Then in `patchMetadataTrack`, why do we want to update the key? Shouldn't we be updating the value of the metadata track instead?

Reply:
Or maybe `metadataTrackKey` is the title of the track column, for example "my awesome_track" in this screenshot below?

<img width="238" alt="Screenshot 2022-04-29 at 09 23 20" src="https://user-images.githubusercontent.com/65414651/165901171-8f24c960-b5f0-4acb-bfb2-0b301a5e41fa.png">

Reply:
yes, this must be it

Reply:
I wonder whether I am the only one that gets confused by metadata track value (the title of the metadata column) vs the metadata track value per sample?

Reply:
I call the title "key" usually to help separate them

Reply:
Sounds good, it just took me some time to figure out. One key with one or more values.